core/src/main/java/org/apache/iceberg/view/ViewMetadataParser.java

```diff
@@ -39,6 +39,7 @@
 
 public class ViewMetadataParser {
 
+  static final String UUID = "uuid";
```

Comment:
for tables this is called `table-uuid`, not sure if we want to align this here and name it `view-uuid`

Reply:
Yes, let's go with `view-uuid`.

Reply:
updated to `view-uuid`